compiler/main/version.cpp

```diff
@@ -35,7 +35,7 @@ void
 get_version(char *v) {
   v += sprintf(v, "%d.%s.%s", MAJOR_VERSION, MINOR_VERSION, UPDATE_VERSION);
   if (strcmp(BUILD_VERSION, "0") != 0 || developer)
-    sprintf(v, ".%s", BUILD_VERSION);
+    sprintf(v, " pre-release (.%s)", BUILD_VERSION);
```

Comment:
I don't think there should be a `.` before the %s once it's parenthesized?

(BTW, not satisfying for today, but I've got a change in progress that should simplify the edits to this file if I ever get around to finishing it up...  this week's been a good reminder to do so).